### PR TITLE
New: Classification link in details sidebar

### PR DESCRIPTION
**Pull request by jeremypress**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jeremypress has signed the CLA. Thanks for the pull request!

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #256

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

File: src/components/ContentSidebar/__tests__/SidebarFileProperties-test.js [hunk 1]
```diff
@@ -3,32 +3,32 @@ import { shallow } from 'enzyme';
...
+                can_rename: true
             }
-        };
+        },
+        onDescriptionChange: jest.fn(),
+        intl: {
+            locale: 'en'
+        }
+    };
+    test('should render the versions when total_count > 0', () => {
```
Comment (DanDeMicco) [inline src/components/ContentSidebar/__tests__/SidebarFileProperties-test.js]: I think this was my fault, but can you correct the test name?

File: src/components/ContentSidebar/__tests__/SidebarFileProperties-test.js [hunk 2]
```diff
@@ -42,15 +42,30 @@ describe('components/ContentSidebar/SidebarFileProperties', () => {
...
         };
-        const wrapper = shallow(<SidebarFileProperties {...props} />).dive();
+        const wrapper = shallow(<SidebarFileProperties {...errorProps} />).dive();
 
         expect(wrapper.find(InlineError)).toHaveLength(1);
         expect(wrapper).toMatchSnapshot();
     });
+
+    test('should render classification information', () => {
+        props.hasClassification = true;
```
Comment (DanDeMicco) [inline src/components/ContentSidebar/__tests__/SidebarFileProperties-test.js]: It doesnt seem like this test does much, looking at the snapshot. I would try using mount, and using enzyme to assert the classification component renders

Comment (jeremypress) [reply src/components/ContentSidebar/__tests__/SidebarFileProperties-test.js]: Updated, let me know if that's what you were thinking.

File: src/components/ContentSidebar/__tests__/SidebarFileProperties-test.js (hunk 1 above)
Comment (jeremypress) [reply src/components/ContentSidebar/__tests__/SidebarFileProperties-test.js]: Updated, thanks

File: src/components/ContentSidebar/__tests__/SidebarFileProperties-test.js (hunk 2 above)
Comment (DanDeMicco) [reply src/components/ContentSidebar/__tests__/SidebarFileProperties-test.js]: Yea I think thats good. Might make sense to have the inverse test as well

**Pull request merged**